src/HeaderAddPlugin.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Http\Client\Plugin;
+
+use Psr\Http\Message\RequestInterface;
+
+/**
+ * Adds headers to the request only if the request does not already own it.
```

Comment:
Should I replace the comment with the following: 

``Adds headers to the request only if the request hasn't it already.`` ?